SDL/BinSkimSetup.ps1

```diff
@@ -0,0 +1,38 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [string]$buildDirectory,
+    
+    [Parameter(Mandatory=$True)]
+    [string]$binSkimDirectory
+)
+
+#$buildDirectory = "C:\Repos\SDK\bin\Release\Src\"
+#$binSkimDirectory = "C:\Repos\SDK\binSkim\"
+
+Write-Host "`nPARAMETERS:";
```

Comment:
Consider using 

    Write-Verbose "Parameters:"

here. Newline not really needed.

Reply:
I understand the purpose of the Verbose distinction, but in this use case I don't think we want this. If a build fails, we would have to run it a second time with the verbose flag to get the details. 
I would rather have the logging always enabled.
Please let me know if you're considering something I'm not thinking of :)